### ISSUE TITLE
Separate backend/front-end pagination in crud settings

### ISSUE THREAD

**Issue by Obversity** [katalyst/koi#340]
At the moment, to turn off pagination on the front-end, you have to do `has_crud paginate: false`. This also turns off pagination in the backend.

**Comment by dbaines** [comment]
This seems to work for me

```
crud.config do
  per_page 10 # this affects front-end
  config :admin
    per_page false # this affects admin
  end
end
```

**Comment by Obversity** [comment]
Yup, I've used that in some places, but that doesn't help with turning off pagination. 

You *can* turn it off by overriding `is_paginated?` in the controller, but that's pretty obscure.

Personally I just feel like pagination on the front-end should not at all be configured in the model, and I've wasted plenty of time trying to work out why the hell things are paginated in the front-end by default before realising/remembering that crud config effects it, and have worked out various not-great ways to override it.
